web/angular-wallet/src/app/main/send-burst/send-burst-form/send-burst-form.component.ts

```diff
@@ -130,8 +131,10 @@ export class SendBurstFormComponent extends UnsubscribeOnDestroy implements OnIn
     this.showMessage = !!this.message;
   }
 
-  getTotal(): number {
-    return parseFloat(this.amount) + parseFloat(this.fee) || 0;
+  getTotal(): BurstValue {
+    return this.amount !== undefined && this.fee !== undefined
+      ? BurstValue.fromBurst(this.amount).add(BurstValue.fromBurst(this.fee))
+      : BurstValue.Zero();
```

Comment:
I like the `fromBurst` and `fromPlanck` methods, removes all ambiguity. Very nice.